airflow/providers/google/cloud/hooks/vertex_ai/batch_prediction_job.py

```diff
@@ -114,6 +114,8 @@ def create_batch_prediction_job(
         labels: dict[str, str] | None = None,
         encryption_spec_key_name: str | None = None,
         sync: bool = True,
+        create_request_timeout: float | None = None,
+        batch_size: int | None = None,
```

Comment:
Since the google API client replace the None value by 64, why we don't provide a 64 as default value to make it clearer (and maybe avoid auto-updating the default value if the client updates it in the next versions)
```suggestion
        batch_size: int = 64,
```

Reply:
Looks like it is their responsibility, do you think would be better duplicate it, or we can trust in Google's implementation? There were a bug once, related to extra copying default value of param

Reply:
IMHO, changing the default value is a breaking change, they will list it in their release notes when they change it, but when we upgrade the client version (like you do in this PR), we don't check all these changes and duplicate them.
I have no objection. @potiuk @pankajkoti WDYT?

Reply:
I had the same question initially, that the docstring says the default is 64, but we set it to None. Later realised that the Google library will set the default to 64 even though we send as None. If we do not want to set the default to 64, IMO, we could still provide more details in the docstring param that the library sets it to 64 even though we set it None. 

If I was developing this, I would set the default to 64 here itself, but that's just my opinion and I maybe wrong :)

Reply:
To be honest I don't think it is a good idea to duplicate existing external code. The description says actual defaults on the final end and we can rely on Google's implementation.

If google would ever change it - we would not need update every wrap of their SDK. But if we will keep default value on our side - we will need to check these points every time when we update of sdk.

Reply:
@potiuk what is your opinion?

P.S. there is another PR (https://github.com/apache/airflow/pull/31082) exact about cutting off default values, what worked fine in the past, but faced with a bug.

Reply:
BTW, [Google's implementation](https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform/jobs.py#L402) has `None` as default value (possibly due to not sending default one over network).

Reply:
Yeah. Having None there is good, and I would say we should simply refer to the default value from the original API - we don't have to specify it here, we can get description "Default is the same as in the BatchPredictionJob from aiplaatform API and we are good.

Honestly - do we care what is the default in this operator in this case?  Not relly, should we describe what it is - yes. Does it have to be a constant? not really - we can defer to the original API. This way it will be both - correct and future-prof.